### PR TITLE
Add adjusted rewards rate

### PR DESCRIPTION
**Pull request by rossbulat**
Adds the adjusted rewards rate.

### CODE REVIEW

File: src/locale/cn/help.json [hunk 1]
```diff
@@ -66,6 +66,13 @@
         "历史年收益",
         ["根据{NETWORK_NAME}奖励分配数据估算的年收益."]
       ],
+      "adjustedRewardsRate": [
+        "Adjusted Rewards Rate",
```
Comment (rossbulat) [inline src/locale/cn/help.json:70]: @TingALin needs translating.

File: src/locale/cn/pages.json [hunk 2]
```diff
@@ -70,6 +70,7 @@
       "activeEra": "活跃 Era",
       "activeNominators": "活跃提名人",
       "addressCopied": "地址已复制到剪贴板",
+      "adjustedRewardsRate": "Adjusted Rewards Rate",
```
Comment (rossbulat) [inline src/locale/cn/pages.json:73]: @TingALin needs translating.

File: src/locale/cn/help.json (hunk 1 above)
Comment (TingALin) [reply src/locale/cn/help.json:70]: "调整后收益率"
[
          "基于｛NETWORK_NAME｝奖励分配模型的估算年收益.",
          "该数字实际上是历史报酬率减去通货膨胀率."
 ]

File: src/locale/cn/pages.json (hunk 2 above)
Comment (TingALin) [reply src/locale/cn/pages.json:73]: "调整后收益率"

**Pull request merged**